### PR TITLE
Added Performace/Utilization card

### PR DESCRIPTION
**Pull request by gnehapk**
ref: https://github.com/kubevirt/web-ui/pull/281

### CODE REVIEW

**Comment by gnehapk** [comment]
@rawagner @mareklibra Please review.

File: src/components/StorageOverview/Utilization/Utilization.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+export const Utilization = ({ iopsUtilization, latencyUtilization, throughputUtilization, LoadingComponent }) => {
+  const iopsStats = getUtilizationVectorStats(iopsUtilization);
+  const latencyStats = getUtilizationVectorStats(latencyUtilization);
+
+  const throughputStatsRaw = getUtilizationVectorStats(throughputUtilization);
+  let throughputStats = null;
+  let throughputMax = 0;
+  let throughputMaxConverted;
+  if (throughputStatsRaw) {
+    throughputMax = Math.max(0, ...throughputStatsRaw);
```
Comment (suomiy) [inline src/components/StorageOverview/Utilization/Utilization.js]: @mareklibra can you please look at the max value here? If it is visualization friendly?

File: src/components/StorageOverview/Utilization/fixtures/Utilization.fixture.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { Utilization } from '../Utilization';
+
+const time0 = 1000;
+const KiB = 1024;
+
+export const utilizationStats = {
+  iopsUtilization: {
```
Comment (rawagner) [inline src/components/StorageOverview/Utilization/fixtures/Utilization.fixture.js]: `latencyUtilization` prop is missing

Comment (gnehapk) [reply src/components/StorageOverview/Utilization/fixtures/Utilization.fixture.js]: Added.

**Pull request merged**